app/views/service_updates/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= render PageTitle::View.new(i18n_key: "service_updates") %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLinkComponent.new(
+    text: t("back"),
+    href: root_path,
+  ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <h1 class="govuk-heading-xl"><%= t(".heading") %></h1>
+
+    <h2 class="govuk-heading-m"><%= t(".contents") %></h2>
+    <ul class="govuk-list govuk-list--dash govuk-!-margin-bottom-8 govuk-!-margin-left-3">
```

Comment:
It doesn't look like the `govuk-list--dash` is rendering anything?

![Screenshot 2021-11-11 at 10 41 49](https://user-images.githubusercontent.com/8417288/141284420-12b3f61c-5b5a-4b9c-81a5-98ed1bcb863c.png)

Reply:
We might need to add our own class here